examples/mobile/UIComponents/components/etc/index.html

```diff
@@ -41,6 +41,11 @@ <h1>
 						Interactive
 					</a>
 				</li>
+				<li class="ui-li-anchor">
+					<a href="version/index.html">
+						Version
```

Comment:
Common practice in apps is to have About page.
How about calling this "About" instead "Version"